store/modules/data/index.js

```diff
@@ -11,8 +11,8 @@ const default_state = {
   // plate_waveforms: 24 element array. each element corresponds to a well in the plate  (all x/y data points for the whole recording)
   plate_waveforms: [],
   heatmap_values: {
-    "Twitch Force": { data: [[0]], range_min: 0, range_max: 100 },
-    "Twitch Frequency": { data: [[100]], range_min: 0, range_max: 100 },
+    "Twitch Frequency": { data: [[1]], range_min: 0, range_max: 1 },
+    "Twitch Force": { data: [[0]], range_min: 0, range_max: 1 },
```

Comment:
Force can probably stay at 100 and I'm wondering about why there are initial data values here? I don't remember why those are there since those values get pushed in through websocket events.

Reply:
I know i  added it about a year ago, but I'm thinking that was a testing mistake that was supposed to be removed

Reply:
are the default values supposed to be there? ^^^